backend/StamAcasa.JobScheduler/JobScheduler.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Cronos;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Timer = System.Timers.Timer;
+
+namespace StamAcasa.JobScheduler
+{
+    internal class JobScheduler : BackgroundService
+    {
+        private readonly IConfiguration _configuration;
+        private readonly ILogger<JobScheduler> _logger;
+        private readonly IServiceScopeFactory _serviceScopeFactory;
+
+        private readonly Dictionary<Type, Timer> _schedule = new Dictionary<Type, Timer>();
+
+        public JobScheduler(
+            IConfiguration configuration,
+            ILogger<JobScheduler> logger,
+            IServiceScopeFactory serviceScopeFactory)
+        {
+            _configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
```

Comment:
If the DI will be able to resolve the services will throw its own errors. There is no need for throw new
Tb sa scriem in engleza? :)